src/components/ControlledSelectInputField/ControlledSelectInputField.js

```diff
@@ -60,15 +79,25 @@ const ControlledSelectInputField = ({
       <Controller
         control={control}
         name={name}
+        rules={rules}
         render={({ value, onChange }) => {
           return (
-            <Select
-              styles={customStyles}
-              options={options}
-              placeholder={placeholder}
-              value={value}
-              onChange={(newValue) => onChange(newValue)}
-            />
+            <>
+              <Select
+                styles={customStyles}
+                options={options}
+                placeholder={placeholder}
+                value={value}
+                onChange={(newValue) => onChange(newValue)}
+              />
+              <Typography
+                variant="caption"
+                display="block"
+                className={classes.errorMsg}
+              >
+                {!!errors?.[name] && errors[name].message}
+              </Typography>
```

Comment:
You need to make sure all the error messages you have are styled the same way, what is the best way t guarantee that?